pkg/e2e/operators/cloudingress/apischeme_cr.go

```diff
@@ -13,35 +15,66 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/client-go/rest"
 )
 
 var _ = ginkgo.Describe(constants.SuiteOperators+TestPrefix, func() {
 	h := helper.New()
 	ginkgo.Context("apischeme", func() {
 		ginkgo.It("apischemes CR instance must be present on cluster", func() {
-			_, err := h.Dynamic().Resource(schema.GroupVersionResource{
-				Group: "cloudingress.managed.openshift.io", Version: "v1alpha1", Resource: "apischemes",
-			}).Namespace(OperatorNamespace).Get(context.TODO(), apiSchemeResourceName, metav1.GetOptions{})
+
+			err := wait.PollImmediate(2*time.Second, 2*time.Minute, func() (bool, error) {
+				if _, err := h.Dynamic().Resource(schema.GroupVersionResource{
+					Group: "cloudingress.managed.openshift.io", Version: "v1alpha1", Resource: "apischemes",
+				}).Namespace(OperatorNamespace).Get(context.TODO(), apiSchemeResourceName, metav1.GetOptions{}); err != nil {
+					return false, nil
+				}
+				return true, nil
+			})
 			Expect(err).NotTo(HaveOccurred())
-		})
 
+		})
+	})
+	ginkgo.Context("apischeme-2", func() {
 		ginkgo.It("dedicated admin should not be allowed to manage apischemes CR", func() {
-			impersonateDedicatedAdmin(h, "test-user")
-			as := createApischeme("apischeme-osde2e-test")
-			err := addApischeme(h, as)
-			Expect(apierrors.IsForbidden(err)).To(BeTrue())
+			user := "test-user"
+			//impersonateDedicatedAdmin(h, user)
+			daGroup, err := h.User().UserV1().Groups().Get(context.TODO(), "dedicated-admins", metav1.GetOptions{})
+			log.Printf("Dedicated admin group 1: %v", daGroup)
+			Expect(err).NotTo(HaveOccurred())
+			daGroup.Users = append(daGroup.Users, user)
+			log.Printf("Dedicated admin group users: %v", daGroup.Users)
+			_, err = h.User().UserV1().Groups().Update(context.TODO(), daGroup, metav1.UpdateOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			time.Sleep(60 * time.Second)
+			testG, _ := h.User().UserV1().Groups().Get(context.TODO(), "dedicated-admins", metav1.GetOptions{})
+			log.Printf("Dedicated admin group: %v", testG)
+
+			//err := wait.PollImmediate(2*time.Second, 2*time.Minute, func() (bool, error) {
```

Comment:
Why is this commented out? Could we just remove this?

Reply:
I was trying things to replicate the errors you were seeing. I updated the code now.